src/SFA.DAS.RoatpAssessor/SFA.DAS.RoatpAssessor.Web/Controllers/Moderator/ModeratorOutcomeController.cs

```diff
@@ -72,14 +76,80 @@ public async Task<IActionResult> ViewOutcome(Guid applicationId)
 
                 return View("~/Views/ModeratorOutcome/Application.cshtml", viewModel);
             }
-            else
+
+
+            var viewModelConfirmation = await _outcomeOrchestrator.GetInModerationOutcomeReviewViewModel(
+                new ReviewModeratorOutcomeRequest(applicationId, userId, command.Status, command.ReviewComment));
+
+            if (command.Status == ModerationStatus.Pass)
+                return View("~/Views/ModeratorOutcome/AreYouSure.cshtml", viewModelConfirmation);
+
+            // this will be removed once APR-1720 and APR-1721 are done
+            return View("~/Views/ModeratorOutcome/AreYouSureHoldingPage.cshtml", viewModelConfirmation);
+
+        }
+
+        [HttpPost("ModeratorOutcomeConfirmation/{applicationId}")]
+        public async Task<IActionResult> SubmitModeratorOutcomeConfirmation(Guid applicationId,  string reviewComment, SubmitModeratorOutcomeConfirmationCommand command)
+        {
+
+            var validationResponse = await _validator.Validate(command);
+            if (validationResponse.Errors.Any())
+            {
+                foreach (var error in validationResponse.Errors)
+                {
+                    ModelState.AddModelError(error.Field, error.ErrorMessage);
+                }
+            }
+
+            var userId = HttpContext.User.UserId();
+
+            if (!ModelState.IsValid)
             {
-                var request =
-                    new ReviewModeratorOutcomeRequest(applicationId, userId, command.Status, command.ReviewComment);
-                var viewModel = await _outcomeOrchestrator.GetInModerationOutcomeReviewViewModel(request);
+                return await GoToErrorView(applicationId, reviewComment, command.Status, userId);
+            }
 
-                return View("~/Views/ModeratorOutcome/AreYouSure.cshtml", viewModel);
+            if (command.ConfirmStatus == "No")
+            {
+                var viewModel = await _outcomeOrchestrator.GetInModerationOutcomeViewModel(new GetModeratorOutcomeRequest(applicationId, userId));
+                viewModel.Status = command.Status;   
+                viewModel.OptionPassText = reviewComment;
+                return View("~/Views/ModeratorOutcome/Application.cshtml", viewModel);
             }
+
+            var userName = HttpContext.User.UserDisplayName();
+
+            var submitSuccessful = await _moderationApiClient.SubmitModerationOutcome(applicationId, userId, userName, command.Status, reviewComment);
+
+            if (!submitSuccessful)
+            {
+                _logger.LogInformation($"Unable to save moderation outcome for applicationId: [{applicationId}]");
```

Comment:
I think this should be a LogError rather than LogInformation as an error has occured and we'll want to see it in Kibana easier :)